.prettierrc.js

```diff
@@ -1,17 +1,16 @@
 module.exports = {
+  // NOTE: This is actually the default value but it is being overwritten by
+  // the solidity plugin somehow.
   bracketSpacing: true,
-  printWidth: 129,
-  semi: false,
```

Comment:
Not using semicolons in JavaScript can lead to ambiguous code. The `prettier` documentation [has a good example outlining this](https://prettier.io/docs/en/rationale.html#semicolons).

Reply:
Fine. As long as we pick one and stay there. I suppose all my contributions to this project have already been overwritten in the annotations anyway so whats another 80% of the entire code base.

Reply:
Again, this only affects TS, JS remains as is. I think the only changed JS files in the diff are the prettier and ESLint cong files